content/en/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md

```diff
@@ -51,7 +51,7 @@ other environment variables get their names from Pod fields.
 Container in the Pod.
 {{< /note >}}
 
-Create the Pod:
+Create the Deployment:
 
```

Comment:
You need to adjust this command: `kubectl logs dapi-envars-fieldref`. That's because there is no longer a Pod named dapi-envars-fieldref. Here's what you need to do:

`kubectl get pods`

The output shows the names of your Pods:

```
NAME                                   READY     STATUS    RESTARTS   AGE
dapi-envars-fieldref-d9bd679b8-dwcjv   1/1       Running   0          16s
dapi-envars-fieldref-d9bd679b8-g6ggt   1/1       Running   0          16s
```

View the logs of the Container that is running in one of your Pods:

`kubectl logs <pod-name>`

where `<pod-name>` is the name of one of your Pods.

Reply:
The same thing applies to this command: `kubectl exec -it dapi-envars-fieldref -- sh`.

It needs to be `kubectl exec -it <pod-name> -- sh`